src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggingServiceCollectionExtensionsTest.cs

```diff
@@ -23,6 +25,43 @@ public void AddLogging_WrapsServiceCollection()
             Assert.True(callbackCalled);
         }
 
+        [Fact]
+        public void AddLogging_InjectScopeProvider()
+        {
+            var services = new ServiceCollection();
+            services.AddSingleton(typeof(IExternalScopeProvider), Mock.Of<IExternalScopeProvider>());
+
+            var callbackCalled = false;
+            var loggerBuilder = services.AddLogging(builder =>
+            {
+                callbackCalled = true;
+                Assert.Same(services, builder.Services);
```

Comment:
This assertion doesn't seem to be adding value and not relevant to injecting scope provider.

Reply:
I have removed the redundant assertion.